User and review dao

Satinder-Sikand opened the pull request:
I have made several commits for the User and Review services for BEAN and DAO. Have also created retrieval functions in the EMartModel which validates parameters and retrieves from DAO.

Satinder-Sikand committed:
Create Review and User foundation

Have created the ReviewDAO and ReviewBean to retrieve reviews. Also created UserDAO and UserBean to retrieve information of a user.

Satinder-Sikand committed:
Add insert functionality for Review and User

Have updated DAO's for Reviews and User classes to include insertion of data for reviews and users into the database tables.

Satinder-Sikand committed:
create retrieval and insert functions in EMartModel

Have created retrieve and insert functions in EMartModel wihich the REST services will use to retrieve data from the DAO services. These functions validate parameters and log errors before attempting to retrieve using DAO.

yash-98 reviewed (changes requested):
Convert _id vars to camelCase

Satinder-Sikand committed:
Removed _ from variables and changed to CamelCase

Changed variable naming style to CamelCase to stay consistent in naming style throughout the project.

yash-98 reviewed (approved):
Changes have been added. PR Approved
